src/Tooltip.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Box from './Box'
+import Card from './Card'
+
+import theme from './theme'
+
+import styled from 'styled-components'
+
+const arrowShadow = props => {
+  return props.top
+    ? {
+        'box-shadow':
```

Comment:
Not sure what lint config this is using but I pasted it into a project that has eslint-config-standard and it's telling me there are 2 extra spaces here.

Reply:
This project doesn't use linting, it uses [prettier](https://github.com/prettier/prettier) to help avoid commenting on code style :)